components/survey/Card/index.jsx

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import Image from 'next/image';
+
+import forwardButton from '@/public/angle-right.svg';
+
+export const surveyCardTestIds = {
+  base: 'survey-card',
+  coverImageUrl: 'survey-card__cover_image_url',
+  title: 'survey-card__title',
+  description: 'survey-card__description',
+  forwardButton: 'survey-card__forward',
+};
+
+const SurveyCard = ({ slideId, title, description, coverImageUrl }) => (
+  <div data-test-id={surveyCardTestIds.base}>
+    <div className="w-[704px] h-[302px] mb-8" data-test-id={surveyCardTestIds.coverImageUrl}>
```

Comment:
Do we have any plan to improve the UI on mobile devices? (this is not a blocker as we don't have the design for mobile)
![Screen Shot 2565-04-22 at 18 10 24](https://user-images.githubusercontent.com/1772999/164703635-5d1fb170-f949-4564-be9c-a4baa7598889.png)

Reply:
currently, I don't have any plan for mobile devices actually
But for this particular page, as the `z-index` is currently overlapping on the slider, I used width in percentage, Thank you!

Reply:
In case there is new screens to be designed, I would recommend trying the [Tailwind breakpoints](https://tailwindcss.com/docs/responsive-design) and using fixed-size width only :) 

- Fixed-size means less "troubles" because everything that fits inside your container (to which the width is applied) will need to be tested only on a specific range of widths. While using percentages requires you to test the whole inifite range of possible widths.
- Breakpoints, only for the sake of being comfortable with them — as on a client project, there is a 99.99% chance that responsiveness will be required (nowadays it is not even a requirement, but an expected assuption xD)

Reply:
Thanks for sharing the link!
I added breakpoints in both `SurveyCard` and the `Sidebar` in 664dcd34571dbbbd6d26d745024c495892f75226